### ISSUE TITLE
Why?

### ISSUE THREAD

**Issue by stdex** [Kingsea442/RainBlog#1]
Hello. It's nice project with nutz, but it's so hard to deploy it...
1) Why you not use maven to build project? it's so easy to download libs from remote repository.
2) Why you use natures from eclipse? It is not quite convenient for other... This is big problem if I want to build project in other IDE (e.g. Intellij IDEA)... I think that the best approach is use deployment to external application server (e.g. tomcat).
3) Why you use direct links in .jsp? It's not correct... e.g. <li><a href="showmodifypassword">修改密码</a></li> - render us http://localhost:8080/showmodifypassword but must be present as http://localhost:8080/admin/showmodifypassword
Turns out enough large connectedness code... If you want to fix something in controller your also must fix something in view (jsp).
4) Where is SQL dump tables structure of the database?

**Comment by Kingsea442** [comment]
Yes, that's good suggestion. And I will study how to use maven. It's just a graduation project, when I have time, I will improve it gradually. Thank you!!!
